### PR TITLE
Feat/livepeer emitter

### PR DESCRIPTION
**Pull request by buildog**
- implementation of livepeerEmitter
- cleaning (remove dist)

### CODE REVIEW

File: app/index-electron.jsx [hunk 1]
```diff
@@ -22,11 +23,16 @@ if (module.hot && isDev) {
...
 const stores = {
-    video: new VideoStore(),
-    win: new WinStore()
+    video: new VideoStore({ events: eventsHandler }),
+    win: new WinStore({ events: eventsHandler })
 };
 
-stores.win.startLivePeer();
+
+setTimeout(() => {
```
Comment (jozanza) [inline app/index-electron.jsx]: Is there some sort of callback or promise that could execute this function?

**Review by jozanza — COMMENTED** [review]
Overall, looks good, but maybe check out my comment  in `app/index-electron.jsx`

File: app/index-electron.jsx (hunk 1 above)
Comment (buildog) [reply app/index-electron.jsx]: probably fixed with all the cleaning I'll give a try and remove it if it works

Comment (jozanza) [reply app/index-electron.jsx]: Right on. Let me know how it goes :)

**Comment by buildog** [comment]
@jozanza reviewed, it was an issue of event registering order, the events was registered at the end of content load which created a race condition.

**Review by jozanza — APPROVED** [review]

**Review by jozanza — COMMENTED** [review]
@buildog awesome job getting rid of that race condition. Putting `startLivepeer` in `WinStore` was a good call.

**Comment by buildog** [comment]
waiting for you review @dob and @ericxtang

**Comment by dob** [comment]
Good by me. Go ahead and merge.

________________________________

**Pull request merged**